userspace/otpilot/src/main.rs

```diff
@@ -69,6 +72,20 @@ fn run_host_helper_demo() -> TockResult<()> {
     Ok(())
 }
 
+fn store_build_info(id: &str, segment_info: SegmentInfo, mut buf: &mut[u8]) {
```

Comment:
Maybe add a comment that `id` is only used for error-spewage?

Reply:
That parameter was actually redundant. I replaced it with code.